lib/assets/AssetsCollection.js

```diff
@@ -52,4 +52,14 @@ AssetsCollection.prototype.asAssetImport = function (name) {
   }, '@import "eyeglass/assets";\n');
 };
 
+/**
+  * Build a string suitable for caching an instance of this
+  * @returns {String} the cache key
+  */
+AssetsCollection.prototype.cacheKey = function(name) {
+  return this.sources.reduce(function(cacheStr, source) {
```

Comment:
should we `sort` `sources` to ensure they remain stable for cacheKey generation?

Reply:
yup, good catch